src/shell.rs

```diff
@@ -15,6 +15,12 @@ pub fn command(root_string: String) -> Result<(), Error> {
     // Build project
     build::main(config, root_path)?;
 
+    // Don't exit on ctrl+c as it is used by child erlang shell
+    ctrlc::set_handler(move || {
+        println!("\nIf you would like to exit this shell, please use Ctrl+g to switch to user switch prompt and type `q` to quit");
+    })
+    .expect("Error setting Ctrl-C handler");
```

Comment:
Let's use `.gleam_expect` here as it provides a nicer error. You'll need to import `crate::error::GleamExpect`.